homeassistant/components/glances/sensor.py

```diff
@@ -299,3 +315,39 @@ def native_value(self) -> StateType:
                 StateType, value[self._sensor_label][self.entity_description.key]
             )
         return cast(StateType, value[self.entity_description.key])
+
+
+class GlancesUptimeSensor(GlancesSensor):
+    """Implementation of a Glances Uptime sensor."""
+
+    _attr_native_value: datetime | None = None
+
+    def __init__(
+        self,
+        coordinator: GlancesDataUpdateCoordinator,
+        description: GlancesSensorEntityDescription,
+        sensor_label: str = "",
+    ) -> None:
+        """Initialize the sensor."""
+        super().__init__(coordinator, description, sensor_label)
+        self._attr_native_value = self.coordinator.get_uptime()
```

Comment:
```suggestion
```
Should be able to remove this completely

Reply:
_handle_coordinator_update is  not called by the DataUpdateCoordinator during initialization of the sensor, so I have to add this initialization of the value for _attr_native_value.
If I don't the sensor remains null until the second API update one minute later.

Reply:
Yes. But in the base class you could have
`self._attr_native_value = self.coordinator.data[self.entity_description.type]`

So instead of calling `get_uptime()` here you can do it in the coordinator to store the data properly in the dict and also it would be better to control the refresh rate of this in the coordinator than in the entity update.

Reply:
Ok got it thanks! 
I'll update this.

Reply:
Done, most of the code specific to the uptime sensor has been moved to the `DataUpdateCoordinator`